### ISSUE TITLE
Head Swivel Issue

### ISSUE THREAD

**Issue by VioletXe** [IgnoredSoul/LethalSex#3]
The head doesn't "tilt" when spinning left again. Works fine when spinning right.

**Comment by IgnoredSoul** [comment]
Yeah, I keep accidentally setting the min to 0 and not negative max. Will be fixed in the next update 👍🏻 
![veqw](https://github.com/IgnoredSoul/LethalSex/assets/61690983/abc02fad-ad23-4244-9fa5-25f2df05a62a)
